tpot/tpot.py

```diff
@@ -288,6 +289,182 @@ def score(self, training_features, training_classes, testing_features, testing_c
 
         return self._evaluate_individual(self.optimized_pipeline_, training_testing_data)[0]
 
+    def export(self, output_file_name):
+        """Exports the current optimized pipeline as Python code.
+
+        Parameters
+        ----------
+        output_file_name: string
+            String containing the path and file name of the desired output file
+
+        Returns
+        -------
+        None
+        """
+        if self.optimized_pipeline_ == None:
+            raise Exception('A pipeline has not yet been optimized. Please call fit() first.')
```

Comment:
I struggled with the proper exception for a bit because the optimized pipeline is technically not a value passed to a parameter. I suppose ValueError is the best fit here if we didn't want to raise a generic Exception.